### PR TITLE
Frame counter (50 fps) now available in G_frames

### PR DESCRIPTION
**Pull request by ConorMcPersonal**
Cheeky one, just adding in that code sample - there's now an unsigned int called G_frames that increments 50 times per second.

### CODE REVIEW

File: SplitFiles/game.c [hunk 1]
```diff
@@ -185,11 +190,15 @@ int main_game()
         music_player->add_music_if_different(music_player, params.effect, 0);
         params.effect = NULL;
       }
+      printf(PRINTAT"\x01\x0B""%d   ", G_frames - last_frame_count);
```
Comment (mcmenaminadrian) [inline SplitFiles/game.c:202]: Do we need this in merged code? I suspect not

File: SplitFiles/game.c [hunk 2]
```diff
@@ -185,11 +190,15 @@ int main_game()
...
   }
   printf(PRINTAT "\x01\x0B" "Final score %d ", (params.slices * params.score));
   bit_beepfx(BEEPFX_AWW);
   //we malloc this so free it
   free(buff.buffer);
+  while (1) {
+    printf(PRINTAT "\x01\x09" "Frame count %d ", (G_frames));
```
Comment (mcmenaminadrian) [inline SplitFiles/game.c]: As above

**Comment by mcmenaminadrian** [comment]
Could you rebase this to current main?

**Review by mcmenaminadrian — COMMENTED** [review]
See comments, take your word that the technical stuff works

*Update* - been a bit cheeky and rebased your PR on main and added some stuff of my own

File: SplitFiles/control.h [hunk 3]
```diff
@@ -6,7 +6,7 @@
 // max stack depth
 static const unsigned int CONTROL_BUFFER_SIZE = 10;
 // length game runs for
-static const unsigned int MAX_TICKS = 8192;
+static const unsigned int MAX_TICKS = 4096;
```
Comment (mcmenaminadrian) [inline SplitFiles/control.h:9]: Gawd it is tedious at 8192 ticks

Comment (ConorMcPersonal) [reply SplitFiles/control.h:9]: LOL it's not a game yet, it's a series of POCs

**Pull request merged**